### PR TITLE
Enable new header in staging

### PR DESCRIPTION
**Pull request by zackkrida**
Enable the new header flag for Openverse on staging only.

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]

**Review by sarayourfriend — APPROVED** [review]

**Comment by sarayourfriend** [comment]
@zackkrida @dhruvkb Do y'all know what the origin of those failed unit tests is? It seems unrelated to this change, at a glance  :confused:

File: feat/feature-flags.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     },
     "new_header": {
       "status": {
-        "staging": "switchable",
+        "staging": "enabled",
```
Comment (obulat) [inline feat/feature-flags.json:13]: We could also probably keep the flag switchable, but set its defaultState to "on" (line 17). This way, we could still go back to the old design in staging using preferences

Comment (obulat) [reply feat/feature-flags.json:13]: Oh, and it's also the only way to get the "old_header" Playwright tests to pass.

Comment (sarayourfriend) [reply feat/feature-flags.json]: I pushed the change you suggested, Olga, so that the PR will be ready for the bug bash later (hopefully).

Comment (obulat) [reply feat/feature-flags.json]: Thank you, @sarayourfriend! I was also working on the adjustments necessary for the tests to pass, and I'll push them in a couple of minutes. It's mostly explicitly disabling the "new_header" flag for the "old" pages.

**Comment by obulat** [comment]
> @zackkrida @dhruvkb Do y'all know what the origin of those failed unit tests is? It seems unrelated to this change, at a glance 😕

Some tests relied on the "new_header" to be off initially.  I've updated them to explicitly set the correct `new_header` flag value.
Some tests were setting the cookie `sameSite` attribute based on the `new_header` flag value, so I updated the expected cookie `sameSite` based on the `new_header` flag value.

**Pull request merged**